pkg/settings/blockchain_settings.go

```diff
@@ -64,6 +64,9 @@ type FunctionalitySettings struct {
 	AddressSchemeCharacter proto.Scheme `json:"address_scheme_character"`
 
 	AverageBlockDelaySeconds uint64 `json:"average_block_delay_seconds"`
+	DelayDelta               uint64 `json:"delay_delta"`
+	// In Milliseconds.
+	MinBlockTime float64 `json:"min_block_time"`
```

Comment:
Add values for these settings to `pkg/settings/embedded/{mainnet,testnet,stagenet}.json` files.